Gruntfile.js

```diff
@@ -344,6 +384,61 @@ module.exports = function(grunt) {
         console.log(`${targetName} extension is in ${dropExtensionPath}`);
     });
 
+    grunt.registerMultiTask('configure-electron-builder', function() {
+        grunt.task.requires('drop:' + this.target);
+        const { dropPath, icon512, fullName, appId, publishUrl } = this.data;
+
+        const outElectronBuilderConfigFile = path.join(dropPath, 'electron-builder.yml');
+        const srcElectronBuilderConfigFile = path.join('src', 'electron', 'electron-builder', `electron-builder.template.yaml`);
+
+        const version = grunt.option('unified-version') || '0.0.0';
+
+        const config = grunt.file.readYAML(srcElectronBuilderConfigFile);
+        config.appId = appId;
+        config.directories.app = dropPath;
+        config.directories.output = `${dropPath}/packed`;
+        config.extraMetadata.version = version;
+        config.publish.url = publishUrl;
+        config.productName = fullName;
```

Comment:
This seems new compared to the `electron-builder.yaml` in master. Docs seem like it might be redundant with extraMetadata.name, but I think it's probably a good idea to add it to be explicit